library/src/morpheus-master/js/src/sakai.morpheus.i18n.utils.js

```diff
@@ -98,6 +99,7 @@
   };
 
   Handlebars.registerHelper('tr', function (namespace, key, options) {
+      console.log("PORTAL TR");
```

Comment:
Can we change it to debug?

Reply:
Thats needs to come out, tbh. Good spot!